### PR TITLE
Document some parts of the Offer contract

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: SampleOfferWithoutReward.sol [hunk 1]
```diff
@@ -17,23 +17,51 @@ along with the DAO.  If not, see <http://www.gnu.org/licenses/>.
...
+    // initial withdraw. The Contractor does not have to do the withdraws every
+    // day as this amount accumulates.
     uint public dailyWithdrawLimit;
 
+    // The address of the Contractor.
     address public contractor;
+
+    // The address of the Proposal/Offer document.
     bytes32 public IPFSHashOfTheProposalDocument;
-    uint public minDailyWithdrawLimit;
```
Comment (LefterisJP) [inline SampleOfferWithoutReward.sol]: you removed a variable here

Comment (chfast) [inline SampleOfferWithoutReward.sol]: Just moved up to have all params set by the Offerer together.

Comment (LefterisJP) [inline SampleOfferWithoutReward.sol]: Right! Good choice. I like it more this way :)

**Comment by LefterisJP** [comment]
Thank you for adding nice docstring and explanations to the contracts @chfast

**Pull request merged**